elrond_dex_farm/src/lib.rs

```diff
@@ -9,6 +9,7 @@ type Nonce = u64;
 const PENALTY_PERCENT: u64 = 10;
```

Comment:
set function for PenalyPercent is need as well.

Reply:
done